pyproject.toml

```diff
@@ -1,7 +1,11 @@
 [tool.ruff]
 # ignore long lines
 ignore = ["E501"]
+select = ["E", "F", "I", "N"]
 
 [tool.mypy]
 ignore_missing_imports = true
 follow_imports = "silent"
+
+[tool.black]
+line-length = 100
```

Comment:
Is this number arbitrary or according to some guideline? In PEP8 it would be 79, in ROOT 120.

Reply:
It's a common setting whenever 80 feels too cramped, which was the case for the AGC code: at least on the two screens I code on, 100 looks better to me than 80 (highly subjective of course).

- PEP8 mentions 100 as another "ok" value if the team agrees: https://pep8.org/#maximum-line-length
- black's default is 88, not 80, because 80 often results in much longer files, and recommends not to go over 100: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Anyway I don't feel strongly about it if you have a preference.